quirks/password-rules.json

```diff
@@ -107,6 +107,9 @@
     "comcastpaymentcenter.com": {
         "password-rules": "minlength: 8; maxlength: 20; required: lower, upper; required: digit; max-consecutive: 2;"
     },
+    "comed.com": {
+        "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; allowed: [-];"
```

Comment:
As far as I can tell, the site allows all special characters other than spaces, meaning the `allowed` character class here could be:

```
[-~!@#$%^&*_+=`|(){}[:;"'<>,.?/\\]]
```

Did you find otherwise?

Reply:
Good point. For some reason, my brain disregarded the last bullet point on that list. I'll get an update pushed. Thanks!